cluster/gce/config-test.sh

```diff
@@ -195,8 +193,10 @@ KUBE_UP_AUTOMATIC_CLEANUP=${KUBE_UP_AUTOMATIC_CLEANUP:-false}
 # is only supported in trusty or GCI.
 TEST_CLUSTER="${TEST_CLUSTER:-true}"
 
-# Storage backend. 'etcd2' supported, 'etcd3' experimental.
+# Storage backend. 'etcd2' and 'etcd3' are supported.
 STORAGE_BACKEND=${STORAGE_BACKEND:-}
+# Storage media type: application/json and application/vnd.kubernetes.protobuf are supported.
```

Comment:
Hmm - I don't know what are the rules, but I didn't see any validation for any of our envs anywhere.
So I think it's not that necessary.